src/test/java/io/logz/LogzioSenderTest.java

```diff
@@ -57,19 +57,19 @@ public void simpleAppending() throws Exception {
         String message1 = "Testing..";
         String message2 = "Warning test..";
 
-        Logger testLogger = createLogger(token, type, loggerName, drainTimeout, null, null, null);
+        Logger testLogger = createLogger(token, type, loggerName, drainTimeout, null, null, null, false, null);
```

Comment:
The last 5 can be null. but almost all calls uses one of the parameters so default values just make it unreadable.. some will have more params, some less